Fix incremental loading of `docker_import`.

mattmoor opened the pull request:
Fixes https://github.com/bazelbuild/rules_docker/issues/79

promiseofcake commented on docker/incremental_load.sh.tpl:
```diff
@@ -24,7 +24,7 @@ DOCKER="${DOCKER:-docker}"
 # Fetch the diff ids of the layers loaded in the docker daemon already
 IMAGES=$(${DOCKER} inspect $(${DOCKER} images -aq) |& \
-  grep sha256 | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
+  grep -E '^ +"sha256:' | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
```
In lieu of requiring `jq` to explicitly select the `Layers` values, this looks like it will work 👍

mattmoor replied:
Yeah, I'd love to depend on `jq`, but I think that's a non-starter for some of the environments this needs to work in :(

dlorenc replied:
What about something like:

```shell
docker inspect $(docker images -aq) -f '{{range .RootFS.Layers}} {{.}} {{end}}' | sort | uniq
```

mattmoor replied:
neat, I'll do that

mattmoor replied:
So that doesn't exactly work as is, but solves our filtering problem if we use:
```
for entry in $(docker inspect $(docker images -aq) -f '{{json .RootFS.Layers}}'); do echo -n $entry | python -mjson.tool; done | grep sha256 | cut -d'"' -f 2 | cut -d':' -f 2 | sort | uniq
```

mattmoor replied:
ok, I updated the PR to use the inspect filtering, nice trick.

mattmoor commented:
lol, I changed the wrong line moving my fix into the Git repo...  As I said in the linked issue, this is relevant to the line:
```
  TOTAL_DIFF_IDS=($(cat "${name}" | python -mjson.tool | \
      grep sha256 | cut -d'"' -f 2 | cut -d':' -f 2))
```

I changed the `docker inspect` line by mistake, oops.

mattmoor commented:
Sadly this means we can't use `inspect` formatting tricks.
